knimin/lib/data_access.py

```diff
@@ -675,15 +682,22 @@ def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
                 # zipcode table & lookup dict
                 info = self.get_geocode_zipcode(zipcode, country)
                 if info.lat is not None:
-                    md[1][barcode]['LATITUDE'] = "%.1f" % info.lat
-                    md[1][barcode]['LONGITUDE'] = "%.1f" % info.long
-                    md[1][barcode]['ELEVATION'] = "%.1f" % info.elev
+                    if full:
+                        md[1][barcode]['LATITUDE'] = info.lat
+                        md[1][barcode]['LONGITUDE'] = info.long
+                        md[1][barcode]['ELEVATION'] = info.elev
+                    else:
+                        md[1][barcode]['LATITUDE'] = round(info.lat, 1)
```

Comment:
This  rounds the newly geocoded data, while the SQL above gets old geocode data that was stored in the DB.